### DISCUSSION TITLE
Do you have discord?

### DISCUSSION THREAD

**Discussion by detherminal** [rozPierog/Cofi#151] [Q&A] [answered]
Do you have personal discord? I would like to chat with you.

**Comment by rozPierog** [comment, marked as answer]
I use Discord only for audio chat while playing games with friends so I'm not that reachable there. I'm not a fan of synchronous communication so I tend to avoid it.
Best way to get in touch is via Mastodon https://fosstodon.org/@LeonOmelan or _shudder_ email that you can find on my GitHub profile.

**Reply by detherminal** [reply]
Hey @rozPierog I emailed to your mail adress in your profile. So can you check it out? Thanks.
